pkg/ui/src/views/statements/statements.spec.tsx

```diff
@@ -292,6 +295,7 @@ describe("selectStatement", () => {
     assert.equal(result.stats.count.toNumber(), sumCount);
     assert.deepEqual(result.app, [stmtA.key.app]);
     assert.deepEqual(result.distSQL, [false, true]);
+    assert.deepEqual(result.opt, [false, true]);
```

Comment:
@knz this is the test case that's failing.  The assertion here says we're expecting to have seen this statement executed with and without `opt` set, but the setup hasn't been updated to create both of them.  It's just making all of them with the default `false`.

Using positional parameters on the `makeFingerprint` helper doesn't really make it obvious, but that's what the four different invocations are doing for the `distSQL` and `failed` keys.